atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/api/WriteReference.java

```diff
@@ -56,13 +56,14 @@ public CellReference cellReference() {
             .registerModule(new Jdk8Module())
             .registerModule(new AfterburnerModule());
 
+    @SuppressWarnings("deprecated")
     public static final Hydrator<WriteReference> BYTES_HYDRATOR = input -> {
         if (input[0] != prefix[0]) {
             return decodeLegacy(input);
         }
         int offset = 1;
         String tableReferenceString = EncodingUtils.decodeVarString(input, offset);
-        TableReference tableReference = TableReference.createFromFullyQualifiedName(tableReferenceString);
+        TableReference tableReference = TableReference.createUnsafe(tableReferenceString);
```

Comment:
I'm not even sure this is right. How about 'fromString'? This class is full of juicy code, e.g.

createUnsafe vs fromString vs fromInternalTableName all having slightly different implementations

Reply:
Do we need to worry about fromInternalTableName and the curse of __ -> ., basically?

Reply:
turns out no, that's a Cassandra thing

Reply:
but still - maybe fromString, since it's not deprecated?